src/styles/blocks/catalog.scss

```diff
@@ -1,3 +1,5 @@
+@import "../utils/_variables.scss";
```

Comment:
There's no need for importing variables here, u can just import them to main.scss.